app/routes/rental_routes.py

```diff
@@ -0,0 +1,111 @@
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import Blueprint, jsonify, request, abort
+from datetime import datetime, timedelta
+from app.routes.video_routes import check_for_valid_input
+
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+def validate_input(request_body):
+    '''ensure all input from request body is a correct 
+    and valid data type, abort and return 400 error 
+    message if the data is not valid'''
+    for key, value in request_body.items():
+        if type(value) is not int:
+            error_message = jsonify({"Invalid Input": f"The {key} value must be an integer."})
+
+            return abort(400, error_message)
+
+@rentals_bp.route("/check-out", methods=["POST"])
+def rental_check_out():
+    
+    request_body = request.get_json()
+
+    list_of_attributes = ["customer_id", "video_id"]
+
+    check_for_valid_input(request_body, list_of_attributes)
+
+    validate_input(request_body)
```

Comment:
to keep your helper functions consistent, maybe call this one `validate_rental_input`